docs/introduction/open-source/public_code.md

```diff
@@ -114,21 +116,22 @@ Requirement | meets | links and notes
 All of the functionality of the codebase – policy as well as source – MUST be described in language clearly understandable for those that understand the purpose of the code. | yes | additional docs are generated from code comments
 The documentation of the codebase MUST contain: a description of how to install and run the source code, examples demonstrating the key functionality. | yes | [getting started](https://open-zaak.readthedocs.io/en/latest/development/getting_started.html), [post-install checklist](https://open-zaak.readthedocs.io/en/latest/installation/index.html#post-install-checklist), improvement: add demo fixtures/instructions
 The documentation of the codebase SHOULD contain: a high level description that is clearly understandable for a wide audience of stakeholders, like the general public and journalists, a section describing how to install and run a standalone version of the source code, including, if necessary, a test dataset, examples for all functionality. |  | [documentation](https://open-zaak.readthedocs.io/en/latest/introduction/index.html), would like to see standard recipies and examples of more fuctionality
-There SHOULD be continuous integration tests for the quality of your documentation. | yes | link checks, build checks, [Travis config](https://github.com/open-zaak/open-zaak/blob/master/.travis.yml#L74)
+There SHOULD be continuous integration tests for the quality of the documentation. | yes | link checks, build checks, [Travis config](https://github.com/open-zaak/open-zaak/blob/master/.travis.yml#L74)
```

Comment:
Link should be updated to the Github Actions, as we migrated from Travis to Github Actions.

Reply:
Added.